app/sidekiq/form0781_state_snapshot_job.rb

```diff
@@ -5,11 +5,17 @@ class Form0781StateSnapshotJob
   include Sidekiq::Job
   sidekiq_options retry: false
 
-  STATSD_PREFIX = 'form0781.state.snapshot'
-  ROLLOUT_DATE = Date.new(2025, 4, 2)
+  STATSD_PREFIX = 'form526.form0781.state.snapshot'
+  STAT_START_DATE = Date.new(2025, 4, 1)
```

Comment:
I'm curious why this particular date, in the past, was chosen.

Reply:
Just a random, somewhat clean (First of a month) date. The date is only there because the job would take too long and time out if it was trying over the full table of in progress forms/submissions.

I have a PR to add metadata to make the search for IPF be much quicker, so may actually merge that first and then this.